pkg/buses/gcppubsub/bus.go

```diff
@@ -284,9 +285,18 @@ func (b *CloudPubSubBus) deleteSubscription(ref buses.SubscriptionReference) err
 }
 
 func (b *CloudPubSubBus) topicID(channel buses.ChannelReference) string {
-	return fmt.Sprintf("channel-%s-%s-%s", b.ref.Name, channel.Namespace, channel.Name)
+	return fmt.Sprintf("channel-%s-%s-%s", channel.Namespace, channel.Name, b.id)
```

Comment:
is it intended to flip the order of this? it use to be general bus to specific channel.